### ISSUE TITLE
Fix the Command in Compiling Docs Section 

### ISSUE THREAD

**Issue by nikhilkalburgi** [wagtail/wagtail#11399]
Change the **cd docs/_build/html/** to **cd _build/html/**

### Pertinent section of the Wagtail docs

(https://docs.wagtail.org/en/stable/contributing/developing.html#compiling-the-documentation)

### Details

The Command in the code snippet produces error in both Windows CMD and GIT Bash because the user will be in the **docs** folder as per the previous command. So, the above change can Fix it.

**Comment by gasman** [comment]
I'm not sure we should assume that these code snippets are always going to be run in sequence - by that reasoning, the following one should start with `cd ../..`, which is probably unhelpful.

It's quite likely that people will want to run the `python -m http.server 8080` command in a separate terminal session, so that they can keep it running while editing and rebuilding the docs - for those people, seeing the full `docs/_build/html` path in the code snippet is useful to help them find the correct location, even if that's not the exact command they'll run.

Maybe we should make this more explicit, and start each snippet with `cd /path/to/wagtail` like the first one? Or add a clarifying comment, like:

```
# starting from the wagtail root directory:
cd docs/_build/html/
python -m http.server 8080
```

**Comment by nikhilkalburgi** [comment]
Hi @gasman,
I totally agree and I think the second option of adding clarifying comment should get executed.
The following changes can be implemented:

```
# Starting from the wagtail root directory:
cd wagtail/
# Install the documentation dependencies
pip install -e .[docs]
# or if using zsh as your shell:
#    pip install -e '.[docs]' -U
# Compile the docs
cd docs/
make html
```

```
# Starting from the wagtail root directory:
cd docs/_build/html/
python -m http.server 8080
```

```
# Starting from the wagtail root directory:
cd docs/
make clean
make html
```

```
# Starting from the wagtail root directory:
cd docs/
make livehtml
```

If you agree, then I can commit these changes into the above PR. Or we can go other way around?

**Comment by anshikavashistha** [comment]
@nikhilkalburgi May I work on this issue?

**Comment by gasman** [comment]
@nikhilkalburgi Yes, that seems good to me - although I think the first snippet should leave out the `cd wagtail/` line (because that would imply going into the `wagtail` subdirectory of the Wagtail codebase).

**Comment by nikhilkalburgi** [comment]
Hi @gasman ,
I have opened a new Pull Request with the approved changes to avoid confusion. Please, have a look.

Thank You
